TRIAGING.md

```diff
@@ -1,6 +1,6 @@
 <img src="https://opensearch.org/assets/img/opensearch-logo-themed.svg" height="64px">
 
-The maintainers of the OpenSearch Repo seek to promote an inclusive and engaged community of contributors. In order to facilitate this, weekly triage meetings are open-to-all and attendance is encouraged for anyone who hopes to contribute, discuss an issue, or learn more about the project. To learn more about contributing to the OpenSearch Repo visit the [Contributing](./CONTRIBUTING.md) documentation.
+The maintainers of the OpenSearch Repo seek to promote an inclusive and engaged community of contributors. In order to facilitate this, weekly triage meetings are open-to-all and attendance is encouraged for anyone who hopes to contribute, discuss an issue, or learn more about the project. There are several weekly triage meetings scoped to the following component areas: Search, Storage, Cluster Manager, and finally "Core" as a catch-all for all other issues. To learn more about contributing to the OpenSearch Repo visit the [Contributing](./CONTRIBUTING.md) documentation.
```

Comment:
sometime we see that issues have more than one component tags. So, based on the proposal, if an issue has one of the above tags and one of catch-all tags, would it be covered in the component meetings or in the both meetings?

Reply:
I would say that whichever the meeting happens first should look at it, and then do one of two things:
- triage the issue, potentially removing the other component tags if they are not applicable
- remove the component tag the current meeting "owns" and leave the issue untriaged so that the other meeting can triage it

What do you think?

Reply:
If I understand correctly, this is what we are ending up doing today. Atleast, thats the case for Cluster Manager.
If the intent is to baseline this across components, then I am good.

However, in the fullness of time, we should see if we can have a mechanism to avoid double triaging. To this, I dont have a good answer at the moment. :genie:

Reply:
@rwali-aws thanks for the thoughts - I'm resolving this comment thread to make progress. 

When we get more data points, it would be great to open a PR to update this document for discussion and adoption.